payments/vipps/after-payment.md

```diff
@@ -243,19 +41,19 @@ The only thing that should be hard coded in the client is the value of the
 the given operation.
 
 {:.table .table-striped}
-| Operation                | Description                                                                                      |
-| :----------------------- | :----------------------------------------------------------------------------------------------- |
-| `update-payment-abort`   | [Aborts][technical-reference-abort] the payment before any financial transactions are performed. |
-| `redirect-authorization` | Used to redirect the consumer to Swedbank Pay Payments and the authorization UI.                 |
-| `create-capture`         | Creates a [`capture`][technical-reference-capture].                                              |
-| `create-cancellation`    | Creates a [`cancellation`][technical-reference-cancel].                                          |
-| `create-reversal`        | Creates a [`reversal`][technical-reference-reverse].                                             |
+| Operation                | Description                                                                                 |
```

Comment:
The links in this table link to Other Features. Should it be linking to the respective operations in After Payment?